config/default.go

```diff
@@ -196,6 +196,22 @@ Specifies the value of ` + "`" + `User-Agent` + "`" + ` header field used for an
 		label:    "true|false",
 		description: `
 Specifies whether a connection to a worker should be reused.  This overrides [the default keep-alive setting](#env-keep-alive).
+`,
+	},
+	"dispatch_max_conns_per_host": {
+		category:     "common",
+		defaultValue: "10",
+		label:        "<number>",
+		description: `
+Specifies maximum idle connections to keep per-host.
```

Comment:
Maybe we should mention that these settings take effect only when `--dispatch-keep-alive` is specified together.